lua/barbar/render.lua

```diff
@@ -152,9 +152,10 @@ local function slice_groups_right(groups, width)
     local text_width = strwidth(group.text)
     accumulated_width = accumulated_width + text_width
 
+    local ellipsis = '…'
```

Comment:
We use ALL_CAPS for constants.

Reply:
Should I take where I yanked this from as incremental refactor then?

https://github.com/tobealive/barbar.nvim/blob/f31c82d6667f503de558b033006f1426ee169dc7/lua/barbar/buffer.lua#L125-L134

`#ellipsis` in the range above resolves to `3`. Which was one of the issued characters that brought me to #414. So `strwidth` might also be appropriate here?

Also constants encapsulated in a module?

Reply:
I'm saying it seems we can take `ellipsis` out of `get_name` and calculate it's length with `strwidth` with the module loads, instead of every time we run this function

Reply:
Thanks for the input @Iron-E. Applied that. In `render.lua` the constants were declared above the related functions. For this module I kept following this pattern.